src/@components/atoms/Button/styles.ts

```diff
@@ -0,0 +1,63 @@
+import { media, styled } from "@styles";
+
+export const Main = styled.button`
+  background-color: ${props => props.theme.colors.turquoise};
+  transform: skew(45deg);
+  padding: 0.9rem 3.7rem;
+  border: none;
+  box-shadow: -5px 5px 14px 0px rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  outline: none;
+  font-family: ${props => props.theme.typography.baseFontFamily};
+  cursor: pointer;
+  color: ${props => props.theme.colors.white};
+
+  &:hover {
+    background-color: ${props => props.theme.colors.turquoiseDark};
```

Comment:
As we discussed this should be something like props.theme.colors.buttonPrimaryHover

Reply:
Agreed